source/localizable/getting-started/quick-start.md

```diff
@@ -211,13 +211,6 @@ curly braces (`{{component}}`). We're going to tell our component:
    provide this route's `model` as the list of people.
 
 ```app/templates/scientists.hbs{-1,-2,-3,-4,-5,-6,-7,+8}
```

Comment:
Hmm, looks like I merged this PR to fix that issue a while back (https://github.com/emberjs/guides/pull/1225) but instead of us fixing the display of the CSS issues we just eliminated `added` / `removed` styles.  <sigh>